install.sh

```diff
@@ -81,17 +81,32 @@ sudo ln -s `pwd`/crew $CREW_PREFIX/bin
 mkdir $CREW_LIB_PATH/lib && cd $CREW_LIB_PATH/lib
 wget -N -c $URL/lib/package.rb
 wget -N -c $URL/lib/package_helpers.rb
-#create the device.json file
+
+#create or update the device.json file
+ruby_version='2.0.0p247'
```

Comment:
On the other hand, it is possible to extract appropriate version number from the name of binary file.  However, their name is already not consistent.  So, I just decide it hard-coded.  ;)